### PR TITLE
Refactor scylladb

### PR DESCRIPTION
**Pull request by rfdickerson**

### CODE REVIEW

File: Sources/TodoListScyllaDB/TodoListError.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+}
+
+extension TodoListError: LocalizedError {
+    var errorDescription: String? {
+        switch(self) {
+        case .creationError(let string):
+            return "Creation error using \(string)"
+        default:
+            return "Some other error"
```
Comment (jesspittman) [inline Sources/TodoListScyllaDB/TodoListError.swift]: I get a compile warning on this line:
```
TodoListError.swift:21:9: warning: default will never be executed
```

**Comment by jesspittman** [comment]
Do you have a version of todolist-tests that works with these changes?

**Comment by rfdickerson** [comment]
Sadly, I don't, actually...

File: Sources/TodoListScyllaDB/TodoListError.swift (hunk 1 above)
Comment (rfdickerson) [inline Sources/TodoListScyllaDB/TodoListError.swift]: Oh, good catch!

File: Sources/TodoListScyllaDB/TodoItemExtension.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import Foundation
+
+
+let localServerURL = "http://localhost:8090"
+
+extension TodoItem : DictionaryConvertible {
+    
+    var url: String {
+        return  "http://127.0.0.1:8090/api/todos/" + docID.uuidString
```
Comment (jesspittman) [inline Sources/TodoListScyllaDB/TodoItemExtension.swift]: This returns the wrong url. Other projects use: 
```
var url: String {
        
        let url: String
        
        do {
            let appEnv = try CloudFoundryEnv.getAppEnv()
            url = appEnv.url
        } catch {
            url = ProcessInfo.processInfo.environment["url"] ?? localServerURL
        }
        
        return url + "/api/todos/" + documentID
    }
```

**Pull request merged**